test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
You are required to git ignore this file. If you git ignore a file, it will not be pushed to your repository.

- [ ] Kindly create a `.gitignore` file (note the dot `.`), add this file name `(test.md)` to it and ensure this file is not pushed to your repo.

- [ ] Add `node_modules` to the `.gitignore` file as well, to ensure it is not pushed to the repo as well.

Reply:
Hi Dammy, the stylelint and webhint keeps showing this error : Process completed at exit code 1. Please help.

Reply:
Only webhint is shown this error: Process completed at exit code 1. Please help.